pkgs/development/python-modules/import-expression/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytest,
+  pytest-cov,
+  astunparse,
+}:
+buildPythonPackage rec {
+  pname = "import-expression";
+  version = "1.1.4";
+  src = fetchFromGitHub {
+    owner = "ioistired";
+    repo = "import-expression-parser";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-mll2NePB7fthzltLOk6D9BgaDpH6GaW4psqcGun/0qM=";
+  };
+  pythonImportsCheck = [
+    "import_expression"
+    "import_expression._codec"
+  ];
+  propagatedBuildInputs = [ astunparse ];
+  nativeCheckInputs = [ pytest pytest-cov ];
+  pytestFlagsArray = [
+    "tests.py"
+    "--cov import_expression"
+    "--cov-config .coveragerc"
+    "--cov-report term-missing"
+    "--cov-report html"
+    "--cov-branch"
+  ];
+
+  # Once https://github.com/ioistired/import-expression-parser/pull/17 gets
+  # merged, remove this.
+  prePatch = ''
+    substituteInPlace tests.py --replace "py.test" "pytest"
+  '';
```

Comment:
Could you please use `fetchpatch` instead?

The patch should be next to `src`.
Otherwise, we might miss patches when we update the source.

Reply:
Sounds like a better idea, sure